preprocessor/folder/folder.go

```
@@ -0,0 +1,84 @@
+package folder
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/rjeczalik/notify"
+	"github.com/starlinglab/integrity-v2/config"
+)
+
+// scanSyncDirectory scans a path under the sync directory and returns a list of files
+func scanSyncDirectory(subPath string) ([]string, error) {
+	scanRoot := config.GetConfig().FolderPreprocessor.SyncFolderRoot
+	if scanRoot == "" {
+		scanRoot = "."
+	}
```

In general I think the service should fail rather than proceed with hidden defaults, let's remove this. It's good to expect the user to configure all these options beforehand. For my own testing I just have a debug version of the config file I use.